File: MapleStoryAvatarSimulator/Alt/ts/avatar-simulator/models/image.ts
import { Map } from "./map";
import { Equipment } from "./equipment";

export class Image {
	constructor(
		public imagePath: string,
		public originX: number,
		public originY: number,
		public zIndex: number,
		public z: string,
		public maps: Map[],
		public ownerEquipment: Equipment
	) {
	}
}

File: MapleStoryDB/Image.cs
using System.Collections.Generic;

namespace MapleStoryDB {
	public class Image {
		public string ImageLogicalPath {
			get;
			set;
		}

		public int? OriginX {
			get;
			set;
		}

		public int? OriginY {
			get;
			set;
		}

		public string Z {
			get;
			set;
		}

		public string Group {
			get;
			set;
		}

		public string Hash {
			get;
			set;
		}

		public string ImageActualPath {
			get;
			set;
		}

		public ICollection<FrameImage> FrameImages {
			get;
			set;
		}

		public ICollection<Map> Maps {
			get;
			set;
		}
	}
}
